InnerLoopAreas.props

```diff
@@ -3,6 +3,5 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Feature to include for inner loop build -->
   <PropertyGroup Condition="$(SolutionName) == 'MUXControlsInnerLoop'">
-    <FeatureRadialGradientBrushEnabled>true</FeatureRadialGradientBrushEnabled>
```

Comment:
can we add all the features here and set everything except one of them to false ? That way you don't have to search for it in the other file and copy it over ?

Reply:
That seems like a good idea. However I am bit concerned if having ~50 lines in there is the best here. After all, all you need to do is just replace the control name. And I think there is barely any control that wouldn't work with the InnerLoop, if there is any control at all.